.github/workflows/release.yaml

```diff
@@ -0,0 +1,56 @@
+name: create release on new tags
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: [ '*.*.*' ]
+      # [0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  build-and-push:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
```

Comment:
I believe we need to setup Go here.

Reply:
To clarify, I believe you need it before you `make bin` in the next step.

Reply:
Good point. Surprisingly in my tests, the `make bin` step ran successfully without having to setup go beforehand. I'll still add this step.